### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by bythewaters**

### CODE REVIEW

**Comment by bythewaters** [comment]
image

image

image

image

File: blog/urls.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+from blog.views import (
+    PostListView,
+    post_detail_view,
+)
+
+urlpatterns = [
+    path("", PostListView.as_view(), name="index"),
+    path(
+        "post/<int:pk>/",
```
Comment (Serhii-Leonenko) [inline blog/urls.py:11]:
```suggestion
        "posts/<int:pk>/",
```

File: blog/views.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        context = {"post": post}
+        return render(request, "blog/post_detail.html", context=context)
+    if request.method == "POST":
+        creation_form = CommentForm(request.POST or None)
+        if creation_form.is_valid():
+            content = request.POST.get("content")
+            comment = Commentary.objects.create(
+                post=Post.objects.get(id=pk),
+                user=request.user, content=content
+            )
```
Comment (Serhii-Leonenko) [inline blog/views.py:29]:
```suggestion
            comment = Commentary.objects.create(
                post=Post.objects.get(id=pk),
                user=request.user, 
                content=content
            )
```

Fix indentation

**Review by Serhii-Leonenko — CHANGES_REQUESTED** [review]

**Review by Serhii-Leonenko — APPROVED** [review]
Good job!

**Pull request closed without merging**